src/commands/install-packages.yml

```diff
@@ -9,7 +9,7 @@ parameters:
     type: enum
     enum: [poetry, pipenv, pip, pip-dist]
     default: pipenv
-    description: Which package management tool to use, pipenv, pip or poetry with dependency file. Use, pip-dist, to install with project setup.py.
+    description: Which package management tool to use, pipenv, pip or poetry with dependency file. Use `pip-dist` to install with project setup.py.
```

Comment:
It doesn't make sense to have commas around this, but I do realize that the backticks are also used inconsistently. Hoping this is just a marginal improvement with room left for more complete standardization.